archive/archive_api/src/bags/fetch_bag.py

```diff
@@ -0,0 +1,22 @@
+# -*- encoding: utf-8
+
+import json
+
+
+def fetch_bag(dynamodb_resource, table_name, s3_client, bucket_name, id):
+    """
+    Fetch the contents of a Bag.
+    """
+
+    table = dynamodb_resource.Table(table_name)
+    item_response = table.get_item(Key={'id': id})
+
+    try:
+        response = s3_client.get_object(
+            Bucket=bucket_name,
+            Key=item_response['Item']['s3key']
```

Comment:
Err, only now realised that #2671 meant a change in the format of stuff stored in the VHS.
Each record has a `location` instead of an `s3Key`. The location is a json object:
```
{    "key" : { "S" : "archive/80/digitised/b22454408/c4efd0b3447783b68ef1dfd5c4acd4499bc5074a86f5a41de6a9b4ebe96df40a" },    "namespace" : { "S" : "wellcomecollection-vhs-archive-manifests" }  }
```
Basically `location` now contains both the key and the bucket name

Reply:
Righto! Working on it...